### ISSUE TITLE
PostgreSQL migration rollback fail

### ISSUE THREAD

**Issue by trulave** [phalcon/phalcon-devtools#1002]
Hi! I want to rollback my database to older version with command:
```
phalcon migration run --version=1.0.0
```
So after that it fails with error:
```
1491055417.9427: DROP TABLE users  => 1491055417.9493 (0.006572961807251)
ERROR: SQLSTATE[42601]: Syntax error: 7 ERROR: syntax error at or near: "LIMIT"
LINE 1: DELETE FROM phalcon_migrations WHERE version='1.0.1' LIMIT 1
```
I was storage my migration history in database. Delete migration [here is](https://github.com/phalcon/phalcon-devtools/blob/master/scripts/Phalcon/Migrations.php#L541) in master branch. But I think there is a syntax error, cause PostgeSQL [doesn't allow limits in delete syntax](https://www.postgresql.org/docs/8.2/static/sql-delete.html).

Sorry for my bad english!

Database: PostgreSQL 9.6
Phalcon version: 3.0.2
Phalcon DevTools version: 3.0.4

**Comment by marcelloh** [comment]
same effect here. The thing is, that the limit should never be there in the first place, since 'it' wants to delete a version (which should be unique) and that deleting all version 1.0.5 is just as good as deleting the first occurrence of version 1.0.5.

**Comment by sergeyklay** [comment]
Fixed in the 3.2.x branch
